repos/system_upgrade/el7toel8/models/repositoriesfacts.py

```diff
@@ -25,3 +26,19 @@ class RepositoriesFacts(Model):
     topic = SystemFactsTopic
 
     repositories = fields.List(fields.Model(RepositoryFile))
+
+
+@deprecated(
+    since="2020-02-01",
+    message=(
+        "The model is temporary and not assumed to be used in any "
+        "other actors."
+    ),
+)
+class TMPTargetRepositoriesFacts(RepositoriesFacts):
```

Comment:
@pirat89 
> add the deprecation and suprress deprecations decorators to make it more clear that this is deprecated to be used by anyone else. The second one should be used properly, so there will not be generated any message in the report about used deprecated model.

Reply:
@pirat89 I don't think the idea with suppress decorator is good. I would like to be notified if our actors use something that is going to be deprecated. Why suppress this?

Reply:
Because we deprecated something intentionally and per the deprecation process we still need to use it. I think it's correct if the warning about using something deprecated is shown only to someone outside of the Leapp team (external contributors, customers). If we don't suppress it, the warning would be shown to **everybody** who uses leapp for the next 6 months. 

The question is how does the **leapp** team get notified that the 6 months period has elapsed and we are good to remove it. This is something we need to figure out, but it's just an internal topic/problem IMHO.

Reply:
this is exactly the reason, why I was thinking that there shouldn't be any difference between the leapp team and everybody. I saw many examples when something deprecated internally and still being raised as a deprecated warning to everyone (pytest, aiohttp of the top of my head...). Anyway, for me was important to bring this thought to the team.

Reply:
Added the suppress decorator in order do not slowdown the process of the review
https://github.com/oamg/leapp-repository/pull/531/commits/5af0c474211dcc05248b507da98e500906eefaca